README.md

```diff
@@ -1,50 +1,31 @@
-## 18F Front End Guild
+## Front End Guild
 
-18F is promoting team best practices across speciality areas through guilds. These guilds support their members in whatever way deemed most appropriate by those members themselves. [This branch](https://github.com/18F/frontend/tree/research) houses guild research into its membership needs.
+18F promotes team best practices across speciality areas through guilds. These guilds support their members in whatever way deemed most appropriate by those members themselves.
```

Comment:
`speciality` is apparently the British spelling of `specialty`

Reply:
ahahahaha care for a cuppa tea?